src/Private/Helpers.ps1

```diff
@@ -3156,4 +3144,181 @@ function Test-PodePlaceholders {
+        } elseif ($Type -ieq 'List`1') {
+            $Type = 'Array'
+        } elseif ($InputObject -is [array]) {
+            $Type = 'Array'
+        } # whatever it thinks it is called
+        elseif ($InputObject -is [hashtable]) {
+            $Type = 'HashTable'
+        } # for our purposes it is a hashtable
+
+        $output += switch ($Type.ToLower()) { 
+            'string' {
+                $String = "$InputObject"
+                if (($string -match '[\r\n]' -or $string.Length -gt 80) -and ($string -inotcontains 'http')) {
+                    $folded = [System.Text.StringBuilder]::new(">`n") # signal that we are going to use the readable 'newlines-folded' format
+                    foreach ($item in $string.Split("`n")) {
+                        $workingString = $item -replace '\r$'
+                        $length = $item.Length
+                        $IndexIntoString = 0
+                        $wrap = 80
+                        while ($length -gt $IndexIntoString + $Wrap) {
+                            $BreakPoint = $wrap
+                            $earliest = $workingString.Substring($IndexIntoString, $wrap).LastIndexOf(' ')
+                            $latest = $workingString.Substring($IndexIntoString + $wrap).IndexOf(' ')
+                            if (($earliest -eq -1) -or ($latest -eq -1)) {
+                                $BreakPoint = $wrap
+                            } elseif ($wrap - $earliest -lt ($latest)) {
+                                $BreakPoint = $earliest
+                            } else {
+                                $BreakPoint = $wrap + $latest
+                            }
+                            if (($wrap - $earliest) + $latest -gt 30) {
+                                $BreakPoint = $wrap # in case it is a string without spaces
+                            }
+
+                            [void] $folded.Append( $padding).AppendLine( $workingString.Substring($IndexIntoString, $BreakPoint).Trim())
+                            $IndexIntoString += $BreakPoint
+                        }
+                        if ($IndexIntoString -lt $length) {
+                            [void] $folded.Append( $padding).AppendLine( $workingString.Substring($IndexIntoString).Trim())
+                        } else {
+                            [void] $folded.AppendLine()
+                        }
+                    }
+                    $folded.ToString()
+                    break
+                } else {
+                    if ($string.StartsWith('#')) {
+                        "'$($string -replace '''', '''''')'"
+                    } else {
+                        $string
+                    }
+                    break
+                }
+                break
+            }
+            'hashtable' {
+                if ($InputObject.Count -gt 0 ) {
+                    $index = 0
+                    <#   foreach ($item in $InputObject.Keys) {
```

Comment:
I need help here.
I'm not able to remove foreach-object 
I tried with the commented code but doesn't work as expected I get an empty line everytime :(